packages/aws-cdk-lib/aws-eks/README.md

```diff
@@ -684,6 +684,25 @@ if (cluster.albController) {
 }
 ```
 
+If you need to configure the underlying ALB Controller, you can pass optional values that will be forwarded to the underling HelmChart construct.
+For example, if your organization uses AWS Firewall Manager you will need to disable aws-load-balancer-controller's waf modification behavior or it will disassociate WAFs that are not specified in the Ingress' annotations.
+
+Note: supported value options may vary depending on the version of the aws-load-balancer-controller helm chart you are using. You should consult the [ArtifactHub documentation](https://artifacthub.io/packages/helm/aws/aws-load-balancer-controller) for your version to ensure you are configuring the chart correctly.
```

Comment:
```suggestion
For example, if your organization uses AWS Firewall Manager you will need to disable `aws-load-balancer-controller`'s waf modification behavior or it will disassociate WAFs that are not specified in the Ingress' annotations.

**Note:** Supported value options may vary depending on the version of the `aws-load-balancer-controller` Helm chart you use. 
Visit the [ArtifactHub documentation](https://artifacthub.io/packages/helm/aws/aws-load-balancer-controller) to ensure you are configuring the chart correctly for your version.
```